src/shared-components/field/__snapshots__/test.tsx.snap

```diff
@@ -114,15 +289,12 @@ exports[`<Field /> UI Snapshot renders with label and labelFor 1`] = `
       className="emotion-2 emotion-3"
       disabled={false}
     />
-    <ul
-      className="emotion-4 emotion-5"
-    />
   </div>
```

Comment:
<ul> is remove cus we return null from VerificationMessage now if message is empty {}